app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```

Comment:
Use gem I18n, move text in file en.yml and translation to vienamese in vi.yml,  Use gem config move "http://www.railstutorial.org/" to file settings